exporter/dorisexporter/exporter_traces.go

```diff
@@ -0,0 +1,255 @@
+    INDEX idx_timestamp(timestamp) USING INVERTED,	
+    INDEX idx_trace_id(trace_id) USING INVERTED,	
+    INDEX idx_span_id(span_id) USING INVERTED,	
+    INDEX idx_trace_state(trace_state) USING INVERTED,	
+    INDEX idx_parent_span_id(parent_span_id) USING INVERTED,	
+    INDEX idx_span_name(span_name) USING INVERTED,	
+    INDEX idx_span_kind(span_kind) USING INVERTED,	
+    INDEX idx_end_time(end_time) USING INVERTED,	
+    INDEX idx_duration(duration) USING INVERTED,	
+    INDEX idx_span_attributes(span_attributes) USING INVERTED,	
+    INDEX idx_status_message(status_message) USING INVERTED,	
+    INDEX idx_status_code(status_code) USING INVERTED,	
+    INDEX idx_resource_attributes(resource_attributes) USING INVERTED,	
+    INDEX idx_scope_name(scope_name) USING INVERTED,	
+    INDEX idx_scope_version(scope_version) USING INVERTED	
+)	
+ENGINE = OLAP	
+DUPLICATE KEY(service_name, timestamp)	
+PARTITION BY RANGE(timestamp) ()	
+DISTRIBUTED BY HASH(trace_id) BUCKETS AUTO	
+%s;	
+`
+)
+
+// dTrace Trace to Doris
+type dTrace struct {
+	ServiceName        string         `json:"service_name"`
+	Timestamp          string         `json:"timestamp"`
+	TraceID            string         `json:"trace_id"`
+	SpanID             string         `json:"span_id"`
+	TraceState         string         `json:"trace_state"`
+	ParentSpanID       string         `json:"parent_span_id"`
+	SpanName           string         `json:"span_name"`
+	SpanKind           string         `json:"span_kind"`
+	EndTime            string         `json:"end_time"`
+	Duration           int64          `json:"duration"`
+	SpanAttributes     map[string]any `json:"span_attributes"`
+	Events             []*dEvent      `json:"events"`
+	Links              []*dLink       `json:"links"`
+	StatusMessage      string         `json:"status_message"`
+	StatusCode         string         `json:"status_code"`
+	ResourceAttributes map[string]any `json:"resource_attributes"`
+	ScopeName          string         `json:"scope_name"`
+	ScopeVersion       string         `json:"scope_version"`
+}
+
+// dEvent Event to Doris
+type dEvent struct {
+	Timestamp  string         `json:"timestamp"`
+	Name       string         `json:"name"`
+	Attributes map[string]any `json:"attributes"`
+}
+
+// dLink Link to Doris
+type dLink struct {
+	TraceID    string         `json:"trace_id"`
+	SpanID     string         `json:"span_id"`
+	TraceState string         `json:"trace_state"`
+	Attributes map[string]any `json:"attributes"`
+}
+
+type tracesExporter struct {
+	*commonExporter
+}
+
+func newTracesExporter(logger *zap.Logger, cfg *Config) (*tracesExporter, error) {
+	commonExporter, err := newExporter(logger, cfg)
+	if err != nil {
+		return nil, err
+	}
+
+	return &tracesExporter{
+		commonExporter: commonExporter,
+	}, nil
+}
+
+func (e *tracesExporter) start(ctx context.Context, _ component.Host) error {
+	if !e.cfg.CreateSchema {
+		return nil
+	}
+
+	conn, err := createMySQLClient(e.cfg)
```

Comment:
This is not a connection to mysql. doris supports mysql protocol and http protocol. http stream load can load data faster, but it cannot create a table. It needs to connect to doris using mysql driver to create a table.

Reply:
Could you just have a README entry explaining how to do it without involving the collector? I'm trying to understand the coupling.

Reply:
Maybe the name of this function should be `createDorisClient`, which using mysql golang driver to connect doris. 
If a user want to create the table manually, the config option `CreateSchema` should be set to false and other config options related to creating tables can be omitted. 
Otherwise, the collector will create the tables automatically. 
I will explain this in README.